PhotoPoints/app/src/main/res/layout/viewholder_photopoint.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="horizontal"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <com.google.android.material.card.MaterialCardView
```

Comment:
I think this should be the "Root" instead of the LinearLayout. The Linearlayout outside of this does not really do anything. I tried, and the result is the same. It's cleaner that way, but not a requirement...

Reply:
That was following an example.  It seemed weird to me, too, so I had already planned to experiment with it.

Reply:
If it was the example we created in the study group, then that's because the setup was like this:
- Linear Horizontal
-- Image
-- Linear Vertical
--- properties

But I understand the confusion. If you would like help with this, let me know!

Reply:
It was an online example.  The cardview was not the root element, nor was it MaterialCardView.  It was simple to move it outside and make it the root.